docs/integrating-with-relay.md

```diff
@@ -0,0 +1,179 @@
+
+Actions come together in Workflows, YAML code written to accomplish the task you're faced with. Workflow authoring is covered in the [Using workflows](./using-workflows.md) documentation, so here we'll focus on creating and using Step and Trigger Actions.
+
+## Creating Actions
+
+In its simplest form, a Relay action is a container image which runs, does some work, and then exits. It's possible to use any OCI-compliant container, as long as its entrypoint terminates with a `0` exit code upon success and a non-zero code upon failure. However, beyond "hello world" examples you'll likely want an image with more capabilities than just using `alpine:latest`. There are two possible paths here, depending on your use case.
+
+1. If the work you're trying to do can be written in a shell or Python script, use the `relaysh/core` image. This is an Alpine-based image that the Relay team maintains, which comes pre-loaded with the Relay SDK. There are `relaysh/core:latest` and `relaysh/core:latest-python` flavors available. You can use these either as a base image in your own Dockerfile or in workflows directly, by specifying an `inputFile` tag whose value is a URL to your script.
+2. If there's an existing image that's made for your integration target, you can use it as the starting point for a custom container. Adding the Relay tools in at build time will allow you to use the metadata service via either the command-line interface or code SDK.
+
+### Using relaysh/core
+
+Using the `relaysh/core` image is the easiest option because it avoids having to build and push a custom container image, but it restricts you to providing your script in a single file. To use it, make your script accessible via https — github repositories or gists both work fine — and provide the URL to it in the `inputFile` attribute on a `step` or `trigger` definition.
+
+There are two variants of `relaysh/core`, indicated by their tag: `relaysh/core:latest` accepts a Bash shell as input and `relaysh/core:latest-python` (as you might expect) expects a Python script. Specify which you want to use in the `image` attribute of your definition.
+
+#### relaysh/core Shell Example
+
+Because webhook Trigger containers need to handle a web request, the shell image isn't suitable for trigger actions, only steps.
+
+```yaml
+steps:
+  - name: run-my-script
+    image: relaysh/core:latest
+    inputFile: https://git.io/JfiwE
+```
+
+#### relaysh/core Python Example
+
+The Python image is usable for both steps and triggers. The following workflow receives a pushed tag from dockerhub and sends a notification with the name of the tag.
+
+```yaml
+parameters:
+  - name: dockerTagName
+    description: The tag of the container which was pushed
+triggers:
+  - name: my-python-trigger
+    source:
+      type: webhook
+      image: relaysh/core:latest
+      inputFile: https://git.io/JfiwD  # TODO this won't work because it imports Quart
```

Comment:
Not sure if you may just want to point this at an existing trigger for now.

Reply:
this works now since i included quart in the core image